### PR TITLE
Created my profile : AbhayKumarMittal.json

### PR DESCRIPTION
**Pull request by MitAbhay**
This PR creates my new profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3278"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Hey Abhay, make your filename same as your GitHub username, otherwise it wont work
Make it `MitAbhay.json`

File: data/AbhayKumarMittal.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Abhay Kumar Mittal",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source | Devops | Full Stack Developer,
```
Comment (Aadarsh805) [inline data/AbhayKumarMittal.json:5]: and there's a missing quotation mark at the end of line 5, before the comma

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/AbhayKumarMittal.json (hunk 1 above)
Comment (MitAbhay) [reply data/AbhayKumarMittal.json:5]: Sure , I will correct it.

**Review by AbbeyIT — DISMISSED** [review]
LGTM now! Thank you for your contribution.

File: data/MitAbhay.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Abhay Kumar Mittal",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source | Devops | Full Stack Developer",
+  "avatar": "https://avatars.githubusercontent.com/u/76659113?v=4",
```
Comment (Panquesito7) [inline data/MitAbhay.json:6]: This link should be used instead as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/MitAbhay.png",
```

**Review by Panquesito7 — DISMISSED** [review]

File: data/MitAbhay.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/MitAbhay" },
+    { "icon": "FaGithub", "url": "https://github.com/MitAbhay" }
+  ],
+  "links": [
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/MitAbhay",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
```
Comment (Panquesito7) [inline data/MitAbhay.json:23]: Not sure there's a color field for links. 🤔

Comment (kumarsonsoff3) [reply data/MitAbhay.json:23]: I think it's optional for links, but we have configured the color of more popular icons [here](https://github.com/EddieHubCommunity/LinkFree/blob/main/config/icons.json). So not required in this case!

**Comment by eddiejaoude** [comment]
Great collaboration 💪

**Pull request merged**